packages/core/src/App/Containers/RealAccountSignup/financial-details.jsx

```diff
@@ -93,108 +94,109 @@ const TradingExperience = ({
         />
     </React.Fragment>
 );
-class FinancialDetails extends React.Component {
-    handleCancel = values => {
-        this.props.onSave(this.props.index, values);
-        this.props.onCancel();
+
+const FinancialDetails = props => {
+    const padding_bottom = window.innerHeight < 930 ? '10rem' : '12rem';
```

Comment:
we are using the number `930` in other components as well, can we move this to a e.g. a descriptive sharable constant?

Reply:
Yes.